### ISSUE TITLE
[RFC]: consider supporting a dark theme

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/www#24]
### Description

This RFC proposes supporting a dark theme for the API docs.

The general idea behind dark theme support to reduce the number of bright pixels (and thus blue light) for better long-term eye health.

This should be straightforward to do. Would mainly require adding a toggle element which would apply a theme class to the root element. From there, we'd just need to update the stylesheets to use different colors (syntax highlighting, links, etc) on the dark background.

### Related Issues

No.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

**Comment by Planeshifter** [comment]
Toggling between a `dark` and `light` theme is meanwhile supported.

**Comment by kgryte** [comment]
This has been added with support with future extension of, e.g., high contrast themes (see GitHub theme preferences). Currently, we only support light and dark.
